docs/configuration.md

```diff
@@ -76,10 +76,10 @@ fladle {
     smartFlankGcsPath = "gs://tmp_flank/flank/test_app_android.xml"
     configs {
         oranges {
-            useOrchestrator = false
-            testTargets = [
+            useOrchestrator.set(false)
```

Comment:
thanks for updating the documentation. I believe that this change is not necessary, but we should make this change in the kotlin example in the docs.

Reply:
https://github.com/runningcode/fladle/pull/176#discussion_r493391055